Dockerfile

```diff
@@ -3,12 +3,15 @@ FROM rust:latest as builder
 RUN USER=root cargo new --bin load-test-server
 WORKDIR /load-test-server
 
+RUN apt-get update
+RUN apt-get install -y libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libgstreamer-plugins-bad1.0-dev gstreamer1.0-plugins-base gstreamer1.0-plugins-good gstreamer1.0-plugins-bad gstreamer1.0-plugins-ugly gstreamer1.0-libav gstreamer1.0-tools gstreamer1.0-x gstreamer1.0-alsa gstreamer1.0-gl gstreamer1.0-gtk3 gstreamer1.0-qt5 gstreamer1.0-pulseaudio
```

Comment:
This is missing `gstreamer1.0-doc` which was recommended in the gstreamer [docs](https://gstreamer.freedesktop.org/documentation/installing/on-linux.html?gi-language=c), but wasn't available when running this in this specific base image.